lib/services/channel.js

```diff
@@ -0,0 +1,43 @@
+var Service = require('./service');
+
+/**
+ * Service for interacting with the channel endpoints on the Beam REST API.
+ * @access public
+ * @augments {Service}
+ */
+function ChannelService () {
+    Service.apply(this, arguments);
+}
+
+ChannelService.prototype = new Service();
+
+/**
+ * Retrieves a list of all channels.
+ * @param  {Object} data
+ * @param  {Object} data.page The page of results to get.
+ * @param  {Object} data.limit Number of results per page to retrieve.
+ * @return {Promise}
```

Comment:
I saw this a few times, it would be great if you documented the resolved value as well.

`@return {Promise.<Type>}` this helps other when reading your code.